includes/classes/Indexable.php

```diff
@@ -317,7 +317,24 @@ public function bulk_index( $object_ids ) {
 			$body .= "\n\n";
 		}
 
-		return Elasticsearch::factory()->bulk_index( $this->get_index_name(), $this->slug, $body );
+		$result = Elasticsearch::factory()->bulk_index( $this->get_index_name(), $this->slug, $body );
+		$return = $result;
+
+		if ( is_wp_error( $result ) ) {
+			$return = false;
```

Comment:
What benefit is there to changing this to `false`, rather than sending the WP_Error to the action?

Also, I think we probably _don't_ want the `ep_after_bulk_index` action to fire in the case of error because that could be confusing to people building on top of it that might be expecting it to only fire after success.

Reply:
Just mimicing ElasticPress functionality, specifically the other after index filters: https://github.com/Automattic/ElasticPress/blob/develop/includes/classes/Elasticsearch.php#L134-L164

If we want to break from that, it might be harder to get it upstreamed.

Reply:
Hmm ok, i guess it makes sense to follow that same pattern then...